nn_gpu/gles/gles_cs_executor.cpp

```diff
@@ -179,6 +179,8 @@ void GlesCsExecutor::showOperationTimers()
         }
         sum += avg;
     }
+    if(sum == 0.0)
```

Comment:
please the coding style in current file, need add space after if

Reply:
Thanks! Already update :=)